rust/cubesql/cubesql/src/compile/engine/df/scan.rs

```diff
@@ -446,8 +446,11 @@ struct CubeScanExecutionPlan {
 }
 
 #[derive(Debug)]
-pub enum FieldValue {
+pub enum FieldValue<'a> {
```

Comment:
Add meaningful name for lifetime, like `'str`/`'string' please

Reply:
I believe it's pretty much the convention to have it as `'a` at least when you only have one lifetime to care about. It's pretty easy to follow with one lifetime.

Reply:
With `Cow<'str, str>` is not an option because it's meme